test/e2e/storage/testsuites/base.go

```diff
@@ -465,3 +480,157 @@ func StartPodLogs(f *framework.Framework) func() {
 
 	return cancel
 }
+
+func getVolumeOpsFromMetricsForPlugin(ms metrics.ControllerManagerMetrics, pluginName string) opCounts {
+	totOps := opCounts{}
+
+	for method, samples := range ms {
+		switch method {
+		case "storage_operation_status_count":
+			for _, sample := range samples {
+				plugin := string(sample.Metric["volume_plugin"])
+				if pluginName != plugin {
+					continue
+				}
+				opName := string(sample.Metric["operation_name"])
+				if opName == "verify_controller_attached_volume" {
```

Comment:
is this emitted for controller metrics?

Reply:
no, but since i've merged the two metrics functions I skip `verify_controller_attached_volume` for both controller and Kubelet metrics. I see no issue with this